assets/components/accessibility/accessibility.scss

```diff
@@ -0,0 +1,8 @@
+.accessibilityHint {
+  position:absolute;
+  left:-10000px;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+}
```

Comment:
As this is tied to the `simpleHeader` component, it might be a better fit in the `simpleHeader.scss` file.

Reply:
It's possible we'll use it in other places. Depending on what else we find :-/